examples/cpp/tutorial.cpp

```diff
@@ -14,14 +14,17 @@ void array_basics() {
   // Get the value out of it:
   auto s = x.item<float>();
   assert(s == 1.0);
+  (void)s;
```

Comment:
C++17 has a `[[maybe_unused]]` for this purpose.
https://en.cppreference.com/w/cpp/language/attributes/maybe_unused.html